rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -147,13 +147,13 @@ impl FmcAliasLayer {
     /// * `Manifest` - Caliptra Image Bundle Manifest
     fn load_manifest(txn: &MailboxRecvTxn) -> CaliptraResult<ImageManifest> {
         let slice = unsafe {
-            let ptr = &mut MAN1_ORG as *mut u8;
-            core::slice::from_raw_parts_mut(ptr, core::mem::size_of::<ImageManifest>())
+            let ptr = &mut MAN1_ORG as *mut u32;
```

Comment:
Why not changing the code in fht.rs?

Reply:
Because afaict that structure isn't being used with the mailbox. Regardless, I would like to change all these globals to be the struct type in some future commit.